### ISSUE TITLE
Czar setting change has no effect

### ISSUE THREAD

**Issue by LadaSobor** [jakelauer/allbadcards#256]
***NOTE***: IF YOU DO NOT PROVIDE DETAILS, YOUR BUG WILL BE CLOSED

BEFORE SUBMITTING:
Please fill out the following to the best of your ability, to help with investigating your report.

1. https://not.allbad.cards/game/mediaeval-rat-2714

2. Whether or not the `Winner Becomes Card Czar` setting is enabled it always gets passed around players, making it impossible to set 1 dedicated czar for the whole session. We have tried from different devices, changing it before starting or during gameplay, but nothing seemed to help.

3. No errors

4. No screenshots.

5. iPhone 11 Pro - Safari, iPhone 8 - Safari, MacBook Air - Google Chrome

**Comment by jakelauer** [comment]
It sounds like what you want is the same as #242 - would you agree?

**Comment by shawntempesta** [comment]
Pretty sure that's what I'm saying. One master czar that determines who gets points throughout.

**Comment by jakelauer** [comment]
Cool - closing in favor of #242
